i18n/locales/de-DE/account.json

```
@@ -9,21 +9,34 @@
         "auth": {
             "login": "Anmelden",
             "register": "Registrieren",
-          "logout": "Abmelden"
+            "logout": "Abmelden"
         },
-      "login": {
-        "email": {
-          "label": "Ihre E-Mail-Adresse",
-          "placeholder": "E-Mail-Adresse",
-          "error": "Bitte geben Sie eine valide E-Mail-Adresse ein"
+        "login": {
+          "email": {
+            "label": "Ihre E-Mail-Adresse",
+            "placeholder": "E-Mail-Adresse",
+            "error": "Bitte geben Sie eine valide E-Mail-Adresse ein"
+          },
+          "password": {
+            "label": "Ihr Passwort",
+            "placeholder": "••••••••",
+            "forgotten": "Passwort vergessen",
+            "errorGeneral": "Bitte geben Sie ein Password ein",
```

```suggestion
            "errorGeneral": "Bitte geben Sie ein Passwort ein",
```

ahhhh i add a typo here! could you adjust it? 🙏🏼